Adding billing metrics (#389)
swan-io/docs: docs/topics/accounts/billing/index.mdx

To learn more, please discuss billing offers with your Account Manager.

## Usage-based billing metrics {#billing-metrics}

> In addition to fixed fees, some Swan features have usage-based fees. The following table lists each billable metric and the event that triggers billing.

Reviewer: ```suggestion
In addition to fixed fees, some Swan features have usage-based fees. You're charged based on actual usage during each billing period.

The following table lists each billable metric and the event that triggers billing.
```
wdyt?

Author: Good detail to add, I like it!

| **Category** | **Metric** | **Billing trigger** |
| --- | --- | --- |

> | **Accounts** | Account subscriptions | Enabled `Verified` accounts not in `Closing` or `Closed` on the 1st of the month. Charged from when the payment level status is set to `Unlimited`. The first and closing months are billed in full. |

Reviewer: ```suggestion
| **Accounts** | Account subscriptions | Accounts with `Verified` status that are `Enabled` and not in `Closing` or `Closed` status on the 1st of each month. Billing begins when the payment level status reaches `Unlimited`. The first and closing months are billed in full. |
```
Clearer?

Author: Much clearer, thanks!

| | Account member subscriptions | All additional `Enabled` members for an account (excluding the legal representative). |
| | Account opening fee | Account holders who started verification during the billing period (`Pending` banking verification status).|
| | Additional account fee | A monthly fee is charged for opening the initial account; this covers the opening of any additional accounts associated with it.|
| **Capital deposit** | Share deposit certificates | Number of capital deposit cases completed during the billing period. |
| **Cards** | Apple Pay card subscriptions | Number of cards successfully added as Apple Pay digital cards during the billing period. |
| | Apple Pay transaction fee | An additional fee for transactions made using Apple Pay. |
| | Card subscriptions | Virtual cards `Enabled` at least one day during the billing period. |
| | Card transactions | Number of card-out booked transactions (including Apple Pay). |

> | | Card products | Number of card products `Enabled` for a project. The monthly fee still applies if a card product is `Suspended`, as long as it has at least one `Enabled` card. |

Reviewer: ```suggestion
| | Card products | Number of card products `Enabled` for a project. Monthly fees still apply to `Suspended` card products that have at least one `Enabled` card. |
```
More concise?
